### PR TITLE
OCPBUGS#11599: simplify storage config example

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OCPBUGS-11599

Link to docs preview:
https://58699--docspreview.netlify.app/microshift/latest/microshift_storage/microshift-storage-plugin-overview.html#microshift-lvmd-yaml-creating_microshift-storage-plugin-overview
NOTE: Netlify seems to be hiccuping today; the preview looks weird.

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@dhellmann PTAL. The preview looks bizarre, I know. I asked Kathryn about it. Let's focus on the content.

File: modules/microshift-lvms-config-example-basic.adoc [hunk 1]
```diff
@@ -16,42 +16,20 @@ The following `lvmd.yaml` example file shows a basic LVMS configuration:
...
-    volume-group: raid-vg
-    lvcreate-options: <9>
-      - --type=raid1
 ----
-<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  `/run/topolvm/lvmd.sock`.
+<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  '/run/lvmd/lvmd.socket'.
 <2> `map[string]DeviceClass`. The `device-class` settings.
 <3> String. The name of the `device-class`.
 <4> String. The group where the `device-class` creates the logical volumes.
 <5> Unit64. Storage capacity in GiB to be left unallocated in the volume group. Defaults to `0`.
```
Comment (dhellmann) [inline modules/microshift-lvms-config-example-basic.adoc:28]: This was probably `uint64` at some point, and "uint" is "unsigned integer".

```suggestion
<5> Unsigned 64-bit integer. Storage capacity in GiB to be left unallocated in the volume group. Defaults to `0`.
```

File: modules/microshift-lvms-config-example-basic.adoc [hunk 2]
```diff
@@ -16,42 +16,20 @@ The following `lvmd.yaml` example file shows a basic LVMS configuration:
...
-    stripe: <7>
-    stripe-size: <8>
-  - name: raid
-    volume-group: raid-vg
-    lvcreate-options: <9>
-      - --type=raid1
 ----
-<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  `/run/topolvm/lvmd.sock`.
+<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  '/run/lvmd/lvmd.socket'.
 <2> `map[string]DeviceClass`. The `device-class` settings.
```
Comment (dhellmann) [inline modules/microshift-lvms-config-example-basic.adoc:25]: "map[string]DeviceClass" makes sense to a go programmer, but may not to a sysadmin.

```suggestion
<2> A list of maps for the settings for each `device-class`.
```

**Review by dhellmann — COMMENTED** [review]
I have couple of suggestions for removing jargon and go syntax. The structure looks great!

**Comment by ShaunaDiaz** [comment]
@dhellmann Are we lgtm?

**Review by dhellmann — COMMENTED** [review]
/lgtm

File: modules/microshift-lvms-config-example-basic.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    spare-gb: 0 <5>
+    default: <6>
+----
+<1> String. The UNIX domain socket endpoint of gRPC. Defaults to  '/run/lvmd/lvmd.socket'.
+<2> A list of maps for the settings for each `device-class`.
+<3> String. The name of the `device-class`.
+<4> String. The group where the `device-class` creates the logical volumes.
+<5> Unsigned 64-bit integer. Storage capacity in GiB to be left unallocated in the volume group. Defaults to `0`.
+<6> Boolean. Indicates that the `device-class` is used by default. Defaults to `false`. At least one value must be entered in the YAML file values when this is set to `true`.
+//Q: still true that value defaults to false?
```
Comment (mburke5678) [inline modules/microshift-lvms-config-example-basic.adoc:30]: @ShaunaDiaz Do you still need an answer to this Q?

Comment (ShaunaDiaz) [reply modules/microshift-lvms-config-example-basic.adoc:30]: @mburke5678 Not for this PR, no.

**Pull request merged**